packages/components/accessible-hidden/src/export-types.ts

```diff
@@ -0,0 +1,3 @@
+import type { TAccessibleHiddenProps as AccessibleHiddenProps } from './accessible-hidden';
+
+export type TAccessibleHiddenProps = AccessibleHiddenProps;
```

Comment:
I have a general remark that even if we add `export-types.ts` file, the types won’t get through to the bundle unless this file is imported in `index.ts` of the package. For instance, with types exported like this there is no possibility of importing types like:
```ts
import AccessibleHidden, {
  type TAccessibleHiddenProps,
} from '@commercetools-uikit/accessible-hidden';
```

You might check e.g. `money-input` for reference.

Reply:
Good point, I will take a look and see how this works now.

Reply:
Circling back to my previous remark, I still think this:
https://github.com/commercetools/ui-kit/blob/5e37083198755a7b94a77d0c4c843e5609d5f2d0/packages/components/inputs/money-input/src/index.ts#L3
needs to be added to each `src/index.ts` to export the types.

Otherwise your `export-types.ts` file gets tree-shaken and `dist/declarations/src/index.d.ts` looks like this:
```ts
// dist/declarations/src/index.d.ts
export { default } from './accessible-hidden';
export { default as version } from './version';
```
While, I think, we expect something like this:
```ts
// dist/declarations/src/index.d.ts
export { default } from './accessible-hidden';
export { default as version } from './version';
export * from './export-types';
```
in the bundle

Reply:
Oh No. Totally cool! 😁 Thanks for the extra eyes 🙌🏾